node-process-argv/echo.js

```diff
@@ -0,0 +1,4 @@
+// console.log('Hello');
+// console.log('Command Line Arguments!');
+// console.log('Hello, computer!');
+console.log('STOP THAT!');
```

Comment:
`echo.js` should output _any_ message that's passed to it, not just the examples, and you shouldn't need to modify the source code each time.

---

![image](https://user-images.githubusercontent.com/16143771/156681543-c366c39b-a76e-4845-a3b4-f74b58986940.png)